modules/using-cli.adoc

```diff
@@ -0,0 +1,221 @@
+// Module included in the following assemblies:
+//
+// * cli/using-roxctl-cli.adoc
+:_module-type: CONCEPT
+[id="using-cli_{context}"]
+= Using the roxctl CLI
+
+Review the following sections to learn how to complete common tasks using the CLI.
+
+[NOTE]
+====
+
+* Export the following variables before using these commands:
++
+[source,terminal]
```

Comment:
Can you put these commands into two separate code blocks? We keep each command in a separate code block because we have a copy button on docs.openshift.com